app/controllers/users.js

```diff
@@ -53,5 +54,18 @@ module.exports = {
         logger.error(`DB: ${error.errors[0]}`);
         res.status(500).json({ error: error.errors[0].message });
       });
+  },
+  usersList(req, res, next) {
+    let page = parseInt(req.query.page) || 1;
+    page = page > 0 ? page : 1;
+    const pageSize = 10;
```

Comment:
the page size should also be a param, with a given default

Reply:
Fixed